src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/AbstractCodeActionOrUserDiagnosticTest.cs

```diff
@@ -25,11 +21,12 @@
 using Xunit;
 
 #if CODE_STYLE
-using EditorConfigFileGenerator = Microsoft.CodeAnalysis.Options.EditorConfigFileGenerator;
-using Microsoft.CodeAnalysis.Internal.Options;
+using System.Diagnostics;
+using System.IO;
+using TestParametersOptions = Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions.IOptionsCollection;
```

Comment:
Not yet because lot of IDE tests still pass in the explicit IDictionary. It will be easy to be done as a follow-up PR that will touch lot of tests, but have purely mechanical changes.